build.gradle

```diff
@@ -7,10 +7,10 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.5.0'
-        classpath 'com.google.gms:google-services:1.4.0-beta1'
+        classpath 'com.google.gms:google-services:1.3.1'
```

Comment:
You cannot update to GMS 8.3.0 because of AppState problem. While I reverting from 1.5.0 I accidentally reverted to 1.3.1. There is not much difference.